paddle/phi/kernels/funcs/concat_and_split_functor.cu

```diff
@@ -19,111 +19,155 @@ limitations under the License. */
 namespace phi {
 namespace funcs {
 
+template <typename T, int Size>
+struct PointerWarpper {
```

Comment:
Is it a typo? Is it a ```PointerWrapper```?

Reply:
oh....it is a typo, will be fixed in next commit.